active/Win32/Window_and_Application_API_Spec.md

```diff
@@ -0,0 +1,618 @@
+</Window>
+```
+```CS
+public partial class MainWindow:Window
+{
+  public MainWindow()
+  {
+	InitializeComponent();
+  }
+}
+```
+##### Multiple Widndows Sample
+Developers can create multi-window experiences with WinUI 3 in the same UI thread. 
+```CS
+Window w = new Window ();
+Button b = new Button() { Content = "Click Me" };
+b.Click += OnClick;
+w.Content = b;
+w.Activate();
+```
+```CS
+private void OnClick(object sender, RoutedEventArgs e)
+{
+   Window w2 = new Window ();
+   w2.Content = new TextBlock(){Text=”Window 2”}
+   w2.Activate();
+}
+```
+##### Create a Window in a separated thread
+To create a Window in a separated thread, you should create the thread first and create a Window in this new Thread. 
+```CS
+private void Button_Click(object sender, RoutedEventArgs e)
+{
+    Thread thread = new Thread(() =>
+    {
+        Window w = new Window();
+        w.Content = new TextBlock() { Text = "Separated Thread Window" };
+        w.Activate();
+        w.Closed += (sender1, e1) => w.Dispatcher.InvokeShutdown();
+
+        System.Windows.Threading.Dispatcher.Run();
+
+    });
+    thread.SetApartmentState(ApartmentState.STA);
+    thread.IsBackground = true; 
+    thread.Start();
+} 
+```
+
+ #### Remarks
+
+- **Win32**: The constructor creates a top-level Win32 Window (HWnd). A top-level Window is a window that is not a child window or has no parent window. 
+
+- **UWP**: There is already one CoreWindow created and UWP only allows one CoreWindow per thread, as a consequence, the contructor and the XAML Markup throw an exception when running in UWP.
+    - Exception type: TBD
+ 
+    - Exception Message: TBD. Something like "Only one Window per thread is allowed in UWP app model".
+
+### Window.Bounds property
+Gets a Rect value containing the height and width of the application window in logical pixels.
+
+```CS
+public Rect Bounds { get; }
+```
+#### Property value
+
+Rect: A value that reports the height and width of the window.
+
+#### Remarks
+
+- **Win32**: The Window is created with the default size of the Window specify by the Window Shell.
+- **UWP**: Keep the current behavior. To change the size of the CoreWindow in UWP, it’s required to use ApplicationView’s PreferredLaunchViewSize and PreferredLaunchWindowingMode.
+
+### Window.Icon property **[NEW]** 
+Gets or sets a window's icon. For each window, this icon is used in its title bar, its task bar button, and in its ALT+TAB application selection list entry.
+
+```CS
+public ImageSource Icon { get; set; }
+```
+
+#### Property value
+Icon: An ImageSource object that represents the icon.
+
+#### Sample
+```XML
+<Window
+  xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+  xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+  x:Class="Sample.MainWindow"
+    Icon="WinUI3Icon.ico">
+</Window>
+```
+
+#### Remarks:
+- **Win32**: The image is set as an Icon of the Window. 
+- **UWP**: Do nothing. The property value is ignored.
```

Comment:
Does it make sense to have a more unified way to specify the icon between Win32 and UWP? I think it comes from the manifest for UWP.

Reply:
With Live Tiles displaying Icons, it probably makes sense to ensure a single ico file can be used to cover all the uses, without having to duplicate.

Reply:
Icon property was moved out of 2020. We will discuss this when we add this feature in the future.